docs/features/modes.md

```
@@ -0,0 +1,102 @@
+export SHIPLOG_TRUST_1_PRINCIPAL="you@example.com"
+export SHIPLOG_TRUST_1_REVOKED="no"
+export SHIPLOG_TRUST_THRESHOLD=1
+export SHIPLOG_TRUST_COMMIT_MESSAGE="shiplog: trust root v1 (GENESIS)"
+export SHIPLOG_ASSUME_YES=1 SHIPLOG_PLAIN=1
+./scripts/shiplog-bootstrap-trust.sh
+```
+
+2) Distribute allowed signers
+- Install the signer roster on each machine and point Git at it:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)